doc/level1.rst

```diff
@@ -140,7 +142,7 @@ Autocrypt is enabled, ``accounts[addr]`` has the following attributes:
    preference for this account.  This is either ``mutual`` or ``nopreference``.
    This SHOULD default to ``nopreference``.
 
-If Autocrypt is enabled for a given account, the MUA SHOULD allow the
+If ``accounts[addr].enabled`` is ``true``, the MUA SHOULD allow the
```

Comment:
Is it necessary to mention that this setting is pointless while autocrypt isn't even enabled?